composer.json

```diff
@@ -53,19 +56,19 @@
 			"/Tests/"
 		]
 	},
+	"config": {
+		"allow-plugins": {
+			"phpstan/extension-installer": true,
+			"symfony/flex": true
+		},
+		"sort-packages": true
+	},
 	"extra": {
 		"branch-alias": {
 			"dev-master": "2.7.x-dev"
 		},
 		"symfony": {
 			"require": "~4.4|~5.4|^6.3"
```

Comment:
I wonder why normalization is ignoring this value.

Reply:
I guess it is because inside `extra` it could be anything, there is no structure.

IIRC `extra.symfony.require` is used by Symfony Flex to restrict the package version used, like the `SYMFONY_REQUIRE` env var used in tests, so I would say we can remove this part of `composer.json` since those versions are already restricted in each symfony package required in the `composer.json` file.